ios/Classes/Blemulator/SimulatedAdapter.m

```diff
@@ -169,13 +169,30 @@ - (void)logLevel:(NS_NOESCAPE Resolve)resolve
 - (void)servicesForDevice:(NSString * _Nonnull)deviceIdentifier
                   resolve:(NS_NOESCAPE Resolve)resolve
                    reject:(NS_NOESCAPE Reject)reject {
+    // TODO: - Error handling
```

Comment:
Check if device is connected and throw if not

Reply:
Added checking for connected state and throwing if not